docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: Grasa-Django-App
+    network_mode: host
+    ports:
+      - 80:80
+      - 443:443
+  db:
+    image: mysql:latest
```

Comment:
Pinning to a specific version lowers the testing surface by standardizing on a specific version we know works. Since MariaDB 10.1 is available [via the IUS repos](https://ius.io/LifeCycle/#mariadb), I suggest pinning this to `mariadb:10.1`.